test_requirements/base.txt

```diff
@@ -4,5 +4,6 @@ django-appconf
 djangocms-helper>=0.9.1
 https://github.com/divio/django-filer/archive/develop.zip
 djangocms-text-ckeditor
+html5lib<0.99999999
```

Comment:
It's very unfortunate but this app uses djangocms-text-ckeditor for testing which in turn installs the latest html5lib which is not compatible with anything.
So we have to limit html5lib to be less than the last release.